adafruit_framebuf.py

```diff
@@ -223,9 +225,81 @@ def scroll(self):
         """scroll is not yet implemented"""
         raise NotImplementedError()
 
-    def text(self):
+    def text(self, string, x, y, color, *,
+             font_name="font5x8.bin"):
         """text is not yet implemented"""
```

Comment:
This could be removed.

Reply:
oh good catch!

Reply:
ok fixed, please approve the PR if its good!